### ISSUE TITLE
Cells not touching

### ISSUE THREAD

**Issue by FiReTiTi** [instanseg/instanseg#137]
Hi,
we gave InstanSeg a try on Codex images, and there seems to be an unnecessary gap between segmented cells (see attached image).
Is there a parameter to play with to reduce such gap?

<img width="527" height="327" alt="Image" src="https://github.com/user-attachments/assets/2c327dff-8d25-4c67-98ad-c598f5797811" />

**Comment by DarioS** [comment]
That minor gap would not matter for other analyses, would it? Aren't you more concerned about missed cells (red)?
<img width="527" height="327" alt="Image" src="https://github.com/user-attachments/assets/0a85b5a5-0ead-4395-9834-124364499b78" />

**Comment by FiReTiTi** [comment]
Hi,
missing cells are indeed an issue.
Non-touching cells involve loosing the membrane or part of it, but it's the expression location of some markers, so highly problematic for protein analysis.

**Comment by ThibautGoldsborough** [comment]
Hi, was that using QuPath or Python? The gaps between the cells are learnt by the model (this is how the training dataset was annotated), so its not an easy fix. If these gaps are an issue for your analysis I recommend a postprocessing step to dilate the labels. If you are using our python repo you can use the function in `utils.pytorch_utils` called `expand_labels_map`. Hope that helps.

**Comment by FiReTiTi** [comment]
> was that using QuPath or Python?

QuPath

> The gaps between the cells are learnt by the model (this is how the training dataset was annotated), so its not an easy fix.

Ok, then this behavior was learnt.
Indeed, fixing it would involving fixing the training set and retrain the model.

> If these gaps are an issue for your analysis I recommend a postprocessing step to dilate the labels. If you are using our python repo you can use the function in `utils.pytorch_utils` called `expand_labels_map`. Hope that helps.

Thank you for the tips.
